.github/workflows/build-package.yml

```diff
@@ -5,24 +5,25 @@ on:
   workflow_dispatch:
 
 jobs:
-  build-windows:
-    runs-on: windows-latest
-    steps:
-      - uses: actions/checkout@v4
-      - name: Set up Python
-        uses: actions/setup-python@v5
-      - name: Install and Build
-        run: |
-          python -m venv .
-          .\Scripts\activate
-          pip install -e .[win,build]
-          mkdir -p dist\windows
-          .\build_windows_exe.ps1 -NoCodeSign
-      - name: Upload build artifact
-        uses: actions/upload-artifact@v4
-        with:
-          name: windows
-          path: dist\windows\piidigger.zip
+  # 9/29/2024: Uncommented the Windows EXE build in favor the embedded build method
```

Comment:
Since this is still in the git history I'd recommend just removing this.

Reply:
Done